src/org/zaproxy/zap/extension/quickstart/QuickStartPanel.java

```diff
@@ -147,48 +159,65 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		
 		urlSelectPanel.add(this.getUrlField(), LayoutHelper.getGBC(0, 0, 1, 1.0D));
 		urlSelectPanel.add(selectButton, LayoutHelper.getGBC(1, 0, 1, 0.0D));
-		panelContent.add(urlSelectPanel, LayoutHelper.getGBC(1, 1, 3, 0.25D));
+		panelContent.add(urlSelectPanel, LayoutHelper.getGBC(1, y, 3, 0.25D));
 		
-		panelContent.add(this.getAttackButton(), LayoutHelper.getGBC(1, 2, 1, 0.0D));
-		panelContent.add(this.getStopButton(), LayoutHelper.getGBC(2, 2, 1, 0.0D));
-		panelContent.add(new JLabel(""), LayoutHelper.getGBC(3, 2, 1, 0.75D, 0.0D));	// Padding to right of buttons
+		panelContent.add(this.getAttackButton(), LayoutHelper.getGBC(1, ++y, 1, 0.0D));
+		panelContent.add(this.getStopButton(), LayoutHelper.getGBC(2, y, 1, 0.0D));
+		panelContent.add(new JLabel(""), LayoutHelper.getGBC(3, y, 1, 0.75D, 0.0D));	// Padding to right of buttons
 		
 		progressLabel = new JLabel(Constant.messages.getString("quickstart.progress." + AttackThread.Progress.notstarted.name()));
 		panelContent.add(new JLabel(Constant.messages.getString("quickstart.label.progress")), 
-				LayoutHelper.getGBC(0, 3, 1, 0.0D, new Insets(5,5,5,5)));
-		panelContent.add(this.progressLabel, LayoutHelper.getGBC(1, 3, 3, 0.0D));
+				LayoutHelper.getGBC(0, ++y, 1, 0.0D, new Insets(5,5,5,5)));
+		panelContent.add(this.progressLabel, LayoutHelper.getGBC(1, y, 3, 0.0D));
 
 		panelContent.add(new JLabel(Constant.messages.getString("quickstart.panel.proxymsg")), 
-				LayoutHelper.getGBC(0, 4, 5, 1.0D, new Insets(5,5,5,5)));
+				LayoutHelper.getGBC(0, ++y, 5, 1.0D, new Insets(5,5,5,5)));
+
+		extSel = Control.getSingleton().getExtensionLoader().getExtension("ExtensionSelenium");
+		if (extSel != null) {
+			try {
+				openProxiedBrowserByName = extSel.getClass().getMethod("openProxiedBrowserByName", String.class);
+			} catch (Exception e1) {
+				LOGGER.error(e1.getMessage(), e1);

```

Comment:
Should be logged as info or debug level? This will happen if the user does not update the Selenium add-on.